package.json

```diff
@@ -7,7 +7,7 @@
     "@typescript-eslint/utils": "^8.29.1",
     "eslint-plugin-eslint-comments": "^3.2.0",
     "eslint-plugin-github": "^5.1.8",
-    "eslint-plugin-import": "^2.31.0",
+    "eslint-plugin-import": "github:qawolf/eslint-plugin-import#prefer-top-level-if-only-type-imports",
```

Comment:
@denis-sokolov I decided to do a timeboxed attempt at `prefer-top-level-if-only-type-imports` and it was actually easy! https://github.com/qawolf/eslint-plugin-import/commit/4228617b0d9739c0662a9c581507c45a0b17edff

I will try to get it merged upstream, but in the meantime I think we should proceed with referencing our own fork here.

Reply:
Note that referencing a fork only works if the built artifacts are in the repository, which they aren’t in eslint-plugin-import right now.

Reply:
They are there in our fork. I committed them on this branch (though not on the branch for my official PR). I confirmed it works this way in platform.